platform-jobs/samza/qrcode-image-generator/src/main/config/qrcode-image-generator.properties

```diff
@@ -59,3 +59,9 @@ cloud_upload_retry_count=__cloud_upload_retry_count__
 cassandra.lp.connection=__cassandra_lp_connection__
 cassandra.lpa.connection=__cassandra_lpa_connection__
 cassandra.sunbird.connection=__cassandra_sunbird_connection__
+
+# QR Image generation default configurations
+# Thickness of white border(in pixels) around the black border of the qr image
+qr_image_margin=1
+# Spacing(in pixels) between qrcode and text in the qr image
+qr_image_margin_bottom=0
```

Comment:
Was there no default value previously for qr_image_margin_bottom?

Reply:
Earlier, the default value was hard-coded at code level